lms/static/sass/features/_learner-profile.scss

```diff
@@ -297,18 +309,20 @@
 
     .wrapper-profile-section-container-two {
         @include float(left);
-        width: calc(100% - 360px);
+        @include padding-left($baseline);
+        width: calc(100% - 380px);
```

Comment:
Again, fixing up a quick issue with responsiveness on the about me input textfield